handbook/digital-experience/process-for-submitting-and-publishing-articles.md

```diff
@@ -0,0 +1,98 @@
+# Process for submitting and publishing articles
+
+This guide includes all you need to know about Fleet’s editorial process and how articles are handled between submission and publication.
+
+> For help with formatting your article, check out our [article formatting guide](./article-formatting-guide).
+
+### On this page
+
+- [Article DRIs](#article-dris)
+- [Who can publish articles?](#who-can-publish-articles)
+- [Communication](#communication)
+- [Schedule](#schedule)
+- [How to submit an article](#how-to-submit-an-article)
+- [Review process](#review-process)
+- [Pre-publication checklist](#pre-publication-checklist)
+- [After publication](#after-publication)
+- [Related pages](#related-pages)
+
+## Article DRIs
```

Comment:
I know "DRI" is a common term here at Fleet, but it was new to me. I didn't know what it meant. Not sure if this warrants revising since it appears throughout the handbook. Might not be worth the trouble.

Reply:
This is a good point. If this wasn't a header I would suggest linking "DRI" to https://fleetdm.com/handbook/people#directly-responsible-individuals.

For simplicity, though, I'm going to leave it as is. Going forward we can perhaps try to onboard the DRI concept better for new members.